app/views/citizens/income_summaries/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= page_template page_title: t('.page_heading'), template: :form do %>
+
+  <p class="govuk-body"><%= t('.add_all_income') %></p>
+    <ol class="app-task-list">
+
+      <% @legal_aid_application.transaction_types.credits.each_with_index do |tt, index| %>
```

Comment:
Use full names in multi-line expressions (`transaction_type` instead of `tt`):
```ruby
<% @legal_aid_application.transaction_types.credits.each_with_index do | transaction_type, index | %>
```

Reply:
really?  In such a short block?

Reply:
Yes. It's a bad habit to get into in my opinion. It makes the code much harder to read and all it saves is a very small amount of typing.